### PR TITLE
Quick Start: Add progress storage

### PR DESCRIPTION
**Pull request by nheagy**
Refs #9447

This PR adds basic storage of quick start progress.

quick start storage added

_Notes_:
This is just basic core data changes and UI to demonstrate them. Checkmarks UI will come in a future PR. Also, the core data code may be refactored to a new class, for now I've left it as the minimum that will work. Let me know if that sickens you 😉 

To test:
- From blog details, tap quick start
- Tap a tour
- Return to the quick start checklist
- Ensure the tour that was tapped is now struckthrough

@astralbodies can you make sure these Core Data changes aren't terrible?

### CODE REVIEW

**Comment by loremattei** [comment]
Hey! I'm moving the milestone to 10.7 as 10.6 has been cut. Feel free to move it back to 10.6 and to target the release branch if it needs to land there!

**Comment by nheagy** [comment]
@aerych can you take a peek? There's now a conflicting `WordPress 77.xcdatamodel`, but I'd like to wait to resolve that until I hear if my CD design is inherently flawed.

Thanks!

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift [hunk 1]
```diff
@@ -1,28 +1,111 @@
 class QuickStartChecklistView: UITableViewController {
```
[the remark this answers is not in the document]
Comment (nheagy) [reply WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift]: That's a good point!

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift [hunk 2]
```diff
@@ -1,28 +1,111 @@
...
+        self.blogID = blogID
+
+        super.init()
+        loadCompletedTours()
+    }
+
+    func loadCompletedTours() {
+        let context = ContextManager.sharedInstance().mainContext
+        let fetchRequest = NSFetchRequest<QuickStartCompletedTour>(entityName: QuickStartCompletedTour.entityName())
+        fetchRequest.predicate = NSPredicate(format: "blog.blogID = %d", blogID)
```
[the remark this answers is not in the document]
Comment (nheagy) [reply WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift]: This is a good suggestion. Exactly the kind of Core Data-focused feedback I was looking for. Thanks!

File: WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift [hunk 3]
```diff
@@ -1,28 +1,111 @@
...
+            if isCompleted(tour: tour) {
+                cell.completed = true
+            }
             return cell
         }
         return UITableViewCell()
     }
+
+    private enum Keys: String {
+        case quickStartTourProgress = "quickStartTourProgress"
```
[the remark this answers is not in the document]
Comment (nheagy) [reply WordPress/Classes/ViewRelated/Blog/QuickStartChecklistView.swift]: Good eye—that's definitely not being used.

**Comment by nheagy** [comment]
@aerych I basically made all your suggested changes 😀 How's it look now?

**Comment by nheagy** [comment]
Thanks @aerych!

**Pull request merged**